content/3_Silver/Greedy.mdx

```diff
@@ -198,6 +198,22 @@ pw.close();
 
 </JavaSection>
 
+<PySection>
+
+```python
+# read in the input, store the algorithms in a list, algorithms
+algorithms.sort();
+count = 0 # number of minutes used so far
+i = 0
+while count + algorithms[i] <= x:
```

Comment:
array overflow? Is this code tested?

Reply:
IIRC integer overflow isn't a problem for Python.

Reply:
array index out of bounds?

Reply:
cpp and java code also do not check if `i` is out of bounds

Reply:
thanks, I'll fix that. Please help me to check if the Java code is correct

Reply:
Wish there was an actual judge to test this on. I definitely am not able to catch all bugs by sight.